### ISSUE TITLE
NTR: Rubella present in CSF viral culture (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8460]
New term request Rubella present in CSF viral culture (samant_101120005211): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000762
- label: Posiitive rubella CSF culture
- parents: [Positive CSF infectious agent test (HP:0430123)](https://hpo.jax.org/app/browse/term/HP:0430123)
- pmid: 
- def: Growth of rubella virus in a culture derived from cerebrospinal fluid (CSF).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8460
- synonyms:
